qwed_finance/integrations/open_responses.py

```
@@ -276,64 +286,67 @@ def _verify_loan_payment(self, args: Dict[str, Any]) -> VerifiedToolCall:
-        result = f"${float(payment):.2f}"
+        computed_payment = str(payment.quantize(Decimal('0.01'), rounding=ROUND_HALF_UP))
+        result = f"${computed_payment}"
         
         receipt = ReceiptGenerator.create_receipt(
             guard_name="OpenResponses.calculate_loan_payment",
             engine=VerificationEngine.SYMPY,
             llm_output=str(args),
-            verified=True,
+            verified=False,  # Computed, not verified against LLM claim
             computed_value=result,
             formula="PMT = P × [r(1+r)^n] / [(1+r)^n - 1]"
```

Ruff (0.15.12)

[warning] String contains ambiguous `×` (MULTIPLICATION SIGN). Did you mean `x` (LATIN SMALL LETTER X)?

(RUF001)